analytic_engine/src/instance/write.rs

```diff
@@ -312,6 +319,16 @@ impl Instance {
             }
         );
 
+        let worker_id = worker_local.worker_id();
+        let worker_num = space.write_group.worker_num();
+        ensure!(
+            table_data.id.as_u64() as usize % worker_num != worker_id,
```

Comment:
The rule might be changed in the future, can we use a function to check this?

Reply:
ok, i'll fix it tonight